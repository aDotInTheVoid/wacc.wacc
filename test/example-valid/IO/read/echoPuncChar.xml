<unit>
  <main>
    <addr_local>0</addr_local>
    <char>'\0'</char>
    <assign_do/>
    <str>"enter a character to echo"</str>
    <print><kind>str</kind><newline>true</newline></print>
    <addr_local>0</addr_local>
    <read><ty>char</ty></read>
    <local>0</local>
    <print><kind>char</kind><newline>true</newline></print>
  </main>
</unit>