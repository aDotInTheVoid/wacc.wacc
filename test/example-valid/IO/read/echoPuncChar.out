enter a character to echo
z
